allennlp/models/semantic_parsing/wikitables/wikitables_decoder_step.py

```diff
@@ -599,6 +719,10 @@ def _compute_new_states(state: WikiTablesDecoderState,
                 action_embedding = action_embeddings[group_index, action_embedding_index, :]
                 production_rule = state.possible_actions[batch_index][action][0]
                 new_grammar_state = state.grammar_state[group_index].take_action(production_rule)
+                if state.checklist_state is not None:
```

Comment:
And this one because `if state.checklist_state[group_index] is not None`, and you can remove the `else`.